NgangaMaryanne/Amity-maryanne
# Ft load state

#### What does this PR do?
Adds docopt to Amity.
#### Description of Task to be completed?
User should be able to interact with the system by issuing commands. 
#### How should this be manually tested?
Clone repo
run command:
_python run.py -i_
This loads the interactive session with amity.
Issue command
 **_help_** to see available commands.
